### PR TITLE
Update samples

### PR DESCRIPTION
**Pull request by brfrn169**
## Description

This PR updates the samples.

## Related issues and/or PRs

N/A

## Changes made

- Updated the ScalarDB versions.
- For ScalarDB SQL related samples, made them to use ScalarDB Cluster.

## Checklist

- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have updated the documentation to reflect the changes.
- [ ] Any remaining open issues linked to this PR are documented and up-to-date (Jira, GitHub, etc.).
- [ ] Tests (unit, integration, etc.) have been added for the changes.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes in other PRs have been merged and published.

## Additional notes (optional)

N/A

### CODE REVIEW

File: microservice-transaction-sample/build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ subprojects {
     ext {
         grpcVersion = '1.53.0'
         protocVersion = '3.23.1'
-        scalarDbVersion = '3.9.1'
+        scalarDbVersion = '3.13.0'
```
Comment (brfrn169) [inline microservice-transaction-sample/build.gradle:8]: Update the ScalarDB versions.

File: scalardb-sql-jdbc-sample/build.gradle [hunk 2]
```diff
@@ -8,18 +8,11 @@ version '1.0-SNAPSHOT'
...
-            password = project.findProperty("gpr.key") ?: System.getenv("TOKEN")
-        }
-    }
 }
 
 dependencies {
-    implementation 'com.scalar-labs:scalardb-sql-jdbc:3.9.1'
-    implementation 'com.scalar-labs:scalardb-sql-direct-mode:3.9.1'
+    implementation 'com.scalar-labs:scalardb-sql-jdbc:3.13.0'
+    implementation "com.scalar-labs:scalardb-cluster-java-client-sdk:3.13.0"
```
Comment (brfrn169) [inline scalardb-sql-jdbc-sample/build.gradle:15]: For ScalarDB SQL related samples, made them to use ScalarDB Cluster.

**Review by komamitsu — APPROVED** [review]
LGTM, thank you!

**Review by josh-wong — APPROVED** [review]
LGTM! Thank you!🙇‍♂️

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Review by kota2and3kan — APPROVED** [review]
LGTM! Thank you!

FYI: I was able to run these samples by following the documents in https://github.com/scalar-labs/docs-internal-scalardb/pull/173 .

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**